flycheck.el

```diff
@@ -4615,6 +4635,7 @@ symbols in the command."
           (process-put process 'flycheck-checker checker)
           (process-put process 'flycheck-callback callback)
           (process-put process 'flycheck-buffer (current-buffer))
+          (process-put process 'flycheck-cwd  default-directory)
```

Comment:
Would you mind to call this property `flycheck-default-directory` instead of `flycheck-cwd`?  It's just easier to understand if the property name preserves the analogy to `default-directory`.